lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -173,6 +173,28 @@
     memory: 1024
     state: present
     network_mode: awsvpc
+
+# Create Task Definition with Environment Variables and Secrets
+- name: Create task definition
+  ecs_taskdefinition:
+    family: nginx
+    containers:
+    - name: nginx
+      essential: true
+      image: "nginx"
+      environment:
+        - name: "PORT"
+          value: "8080"
+      secrets:
+        - name: "VAR_NAME_1"
+          valueFrom: "arn:aws:secretsmanager:region:aws_account_id:secret:secret_name_for_ENV_NAME_1"
```

Comment:
probably clearer as `secret_name_for_VAR_NAME_1` (same for `VAR_NAME_2` below)

Reply:
Thanks @willthames. Changes committed.